packages/@ama-sdk/client-fetch/src/plugins/abort/abort.fetch.ts

```diff
@@ -0,0 +1,90 @@
+import type { FetchCall, FetchPlugin, FetchPluginContext, RequestOptions } from '@ama-sdk/core';
+
+interface AbortCallbackParameters {
+  /** URL targeted */
+  url: string;
+
+  /** Fetch call options */
+  options: RequestInit | RequestOptions;
+
+  // TODO Now supported for all the modern browsers - should become mandatory in @ama-sdk/core@11.0
```

Comment:
Your PR will be merged in v11.3 or later 
shouldn't we set controller as mandatory for this new package ?